tools/test.py

```diff
@@ -1362,6 +1383,9 @@ def BuildOptions():
   result.add_option('--abort-on-timeout',
       help='Send SIGABRT instead of SIGTERM to kill processes that time out',
       default=False, action="store_true", dest="abort_on_timeout")
+  result.add_option("--actions-annotation",
```

Comment:
Have you considered making the annotations another progress indicator rather than another option?

Reply:
That's a good idea, it could be a progress indicator inheriting from dotted. Lemme give it a try.

Reply:
Done, what do you think?

Reply:
LGTM, thanks.